recipe/meta.yaml

```diff
@@ -1,13 +1,12 @@
-{% set version = "0.15.71" %}
-# The version_info tuple in __init__.py must also be updated
+{% set version = "0.15.80" %}
```

Comment:
`0.15.80` to get https://bitbucket.org/ruamel/yaml/commits/c4f22d89a9e22cb5b8c75daaa5c655fae975c84a but avoid https://bitbucket.org/ruamel/yaml/commits/f3bb7c69afd053ad34701cb68c29d53e509dd4a5.

It is probably fine to go up to `0.15.100`, but see https://github.com/conda-forge/conda-feedstock/pull/81#issuecomment-551423117 and https://github.com/conda-forge/conda-feedstock/pull/81#issuecomment-551975922.